infra/license/license-judgment.json

```diff
@@ -62,5 +62,13 @@
     "(MIT OR CC0-1.0)": {
         "permitted": "conditional",
         "caution": "Non-single license is hard to be judged automatically"
+    },
+
+    "(MIT AND Zlib)": {
```

Comment:
The zlib license is a [permissive free software license](https://en.wikipedia.org/wiki/Permissive_free_software_license) which defines the terms under which the [zlib](https://en.wikipedia.org/wiki/Zlib) [software library](https://en.wikipedia.org/wiki/Library_(software)) can be distributed. It is also used by many other [free software](https://en.wikipedia.org/wiki/Free_software) packages. The [libpng](https://en.wikipedia.org/wiki/Libpng) library uses a similar license sometimes referred interchangeably as zlib/libpng license.

The zlib license has been approved by the [Free Software Foundation](https://en.wikipedia.org/wiki/Free_Software_Foundation) (FSF) as a [free software license](https://en.wikipedia.org/wiki/Free_software_license),[[2]](https://en.wikipedia.org/wiki/Zlib_License#cite_note-fsf-2) and by the [Open Source Initiative](https://en.wikipedia.org/wiki/Open_Source_Initiative) (OSI) as an [open source license](https://en.wikipedia.org/wiki/Open_source_license).[[3]](https://en.wikipedia.org/wiki/Zlib_License#cite_note-osi-3) It is [compatible](https://en.wikipedia.org/wiki/License_compatibility) with the [GNU General Public License](https://en.wikipedia.org/wiki/GNU_General_Public_License).[[2]](https://en.wikipedia.org/wiki/Zlib_License#cite_note-fsf-2)